### PR TITLE
fix(refs:T31024): Space missing by copy from docx WIP

### PR DESCRIPTION
**Pull request by elmasdemos**
**Ticket**: https://yaits.demos-deutschland.de/T31024

**Description**:
- if you copy a text from a word docx into tiptap, some spaces are missing

### CODE REVIEW

File: src/components/core/DpEditor/DpEditor.vue [hunk 1]
```diff
@@ -1153,6 +1153,9 @@ export default {
           // Strip img tags from pasted and dropped content
           returnContent = returnContent.replace(/<img.*?>/g, '')
 
+          // Strip line break spaces, white spaces and new lines
+          returnContent = returnContent.replace(/(&nbsp;|\r|\n)/gmi, '')
```
Comment (spiess-demos) [inline src/components/core/DpEditor/DpEditor.vue:1157]: This looks like all those characters are replaced by the empty string, which leads to all spaces between words being dropped. I guess this is not what we want. I'd suggest to replace the characters with one space character.

```suggestion
          returnContent = returnContent.replace(/(&nbsp;|\r|\n)/gmi, ' ')
```

**Review by spiess-demos — CHANGES_REQUESTED** [review]
Apart from the suggestion i think that the changes in package.json should not be part of this PR.

File: src/components/core/DpEditor/DpEditor.vue (hunk 1 above)
Comment (elmasdemos) [reply src/components/core/DpEditor/DpEditor.vue:1157]: thank you, it thought i put a space in already

Comment (elmasdemos) [reply src/components/core/DpEditor/DpEditor.vue]: done https://github.com/demos-europe/demosplan-ui/pull/80/commits/82f72fb092309951b759dd14d6ad171507ea40e1

**Comment by hwiem** [comment]
Can we close this? The bug was fixed in https://github.com/demos-europe/demosplan-ui/pull/93

**Comment by elmasdemos** [comment]
closed because fixed in https://github.com/demos-europe/demosplan-ui/pull/93

**Pull request closed without merging**